### ISSUE TITLE
Element Catalog Site NET::ERR_CERT_DATE_INVALID

### ISSUE THREAD

**Issue by dman777** [Polymer/polymer-element-catalog#218]
At any coffee shop wifi, for both firefox and chrome, when trying to access https://elements.polymer-project.org/ I get:

```
Attackers might be trying to steal your information from elements.polymer-project.org (for example, passwords, messages, or credit cards). NET::ERR_CERT_DATE_INVALID
ReloadHide advanced
elements.polymer-project.org normally uses encryption to protect your information. When Chrome tried to connect to elements.polymer-project.org this time, the website sent back unusual and incorrect credentials. Either an attacker is trying to pretend to be elements.polymer-project.org, or a Wi-Fi sign-in screen has interrupted the connection. Your information is still secure because Chrome stopped the connection before any data was exchanged.

You cannot visit elements.polymer-project.org right now because the website uses HSTS. Network errors and attacks are usually temporary, so this page will probably work later.
```

I am using `nameserver 8.8.8.8` in my resolve.conf, so I do not think it's DNS.

**Comment by ebidel** [comment]
The error seems to suggest an issue with the shop's WiFi login. Please
verify from another connection when you get a chance.

**Comment by dman777** [comment]
That's what I thought also...and still could be true. However, the same thing happened at 2 different coffee shops that used different ISP's. One has a portal page and the other doesn't, so this is how I am able to guess they are using different ISPs. Never happens at my house though.

**Comment by dman777** [comment]
This issue just recently stopped after a couple of weeks(worked today at the coffee shop) so I will close this issue.

**Comment by dman777** [comment]
Just a follow up...

This happened again to me last night at a coffee shop. I discovered the root cause was my laptop time was wrong(old motherboard) and I used ntp-client to sync the time. After that, I was able to use the polymer elements catalog. 

What is interesting though is that this issue only effected the polymer elements catalog site....other site were not effected and I was not blocked from other sites(including the polymer guide site).
